tests/storage/conftest.py

```
@@ -230,12 +230,14 @@ def uploadproxy_route_deleted(hco_namespace):
 
 @pytest.fixture()
 def cdi_config_upload_proxy_overridden(
+    admin_client,
     hco_namespace,
```

Ruff (0.15.21)

[warning] Unused function argument: `hco_namespace`

(ARG001)